Akash/html/assign 2/Assign2.html

```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <title>
+            Assign2.html
+        </title>
+    </head>
+    <body>
+        <h1>HTML</h1>
+        <h3>Introduction</h3>
+        <p>
+            HTML is a standard markup language for web pages.
+        </p>
+        <h5>Notepad or textedit</h5>
+        <p>Web paged can be created and modified by using professional HTML editors.<br>
+           However,for learning HTML we recommend a simple text editor like Notepad(pc)or textedit(Mac).</p>
+    </body>
+</html> 
+ 
```

Comment:
check is there any trailing spaces present in line no. 19 as its showing the error of no new line at end of file